htdocs/js/options.js

```diff
@@ -92,13 +92,22 @@ vz.options.plot = {
 			tickFormatter: vz.wui.tickFormatter		// show axis label
 		},
 		{
-			axisLabel: '°C', // assign el. energy to first axis- remove if not used
+			axisLabel: '°C', // assign temperature to 2nd axis- remove if not used
 			tickFormatter: vz.wui.tickFormatter		// show axis label
 		},
 		{
 			// alignTicksWithAxis: 1,
 			position: 'right',
 			tickFormatter: vz.wui.tickFormatter		// show axis label
+		},
+		{
+			/*
+			** Please note: The last axis defined in here will also be used as a 
+			** template to clone further axes as needed. All settings (except the 
+			** axis label) will replicate into those additional axes.
+			*/
```

Comment:
Zusätzliche Achsen sollten rechts sein?

Reply:
Nein, ich habe die bisher im default options.js definierten Achsen so gelassen, wie sie waren, und eine zusätzliche definiert, unter dem Kommentar, mit `position: left` Damit verhält sich der VZ bei nicht editiertem options.js genau so wie bisher.

{
			/*
			** Please note: The last axis defined in here will also be used as a 
			** template to clone further axes as needed. All settings (except the 
			** axis label) will replicate into those additional axes.
			*/
			position: 'left',
			tickFormatter: vz.wui.tickFormatter             // show axis label
		}

Reply:
Bisher wird die 2. Achse gecloned- alignment rechts. Jetzt wird die letzte Achse gecloned- die hat aber bei Dir Position left. Ich würde die letzte Achse für gleiches Verhalten rechts erwarten?

Reply:
Hmm, die zweite Achse hat standardmässig keine Position definiert. Wenn ich testweise die default-config bei mir verwende, kommt die Temperatur und auch all die reproduzierten Achsen links. 

Allerdings mag das auch mit den bei mir verwendeten Kanälen zu tun haben... so habe ich z.B. keine Watt, d.h. die erste Achse wird dann ausgeblendet. Da müsste man nochmal genau reinsehen, was das Frontend macht wenn keine Position angegeben ist.

Alternativ könnten wir uns überlegen wie es sein sollte, und es in der Config entsprechend anpassen. Ob die "geclonten" Achsen in der Stardard-Config links oder rechts sind, ist mir prinzipiell völlig egal... kann ja jeder so anpassen wie er/sie möchte.

Reply:
> Hmm, die zweite Achse hat standardmässig keine Position definiert. Wenn ich testweise die default-config bei mir verwende, kommt die Temperatur und auch all die reproduzierten Achsen links.

Die zweite Achse in der Konfig hat `position: right` soweit ich das oben sehe. Geclonte Achsen möchte per Default gerne rechts haben, analog heutigem Verhalten.

Reply:
Nein, das ist die dritte Achse mit Position: Rechts. Die erste und zweite Achse haben keine Position angegeben, und meiner Beobachtung nach setzt flot bei fehlender position alle Achsen links.

Reply:
Mhhm, verstehe. Dann war das vorher schon falsch- egtl. hätte die letzte Achse verwendet werden sollen. Die Templateachse sollte right-aligned sein.

Reply:
Dann könnte man anstelle einer zusätzlichen 4. Achse als Template auch gleich die 3. Achse als Template mit Kommentar versehen, diese Achse hätte ja dann schon die entsprechende Konfiguration - was meinst Du?

Reply:
gerne